docs/reference/docker-compose.md

```diff
@@ -87,15 +87,17 @@ relative to the current working directory.
 
 The `-f` flag is optional. If you don't provide this flag on the command line,
 Compose traverses the working directory and its subdirectories looking for a
-`docker-compose.yml` and a `docker-compose.override.yml` file. You must supply
-at least the `docker-compose.yml` file. If both files are present, Compose
-combines the two files into a single configuration. The configuration in the
-`docker-compose.override.yml` file is applied over and in addition to the values
-in the `docker-compose.yml` file.
+`docker-compose.yml` and a `docker-compose.override.yml` file. See also the
+`COMPOSE_FILE` [environment variable](overview.md#compose_file). You must
```

Comment:
I think this "see also ..." sentence needs to either be in paranthesis, or maybe moved to the end of the paragraph. It's kind of breaking up the flow.